trackbasket_be/resources/shopping_list.py

```diff
@@ -0,0 +1,70 @@
+from flask_restful import Resource, reqparse, request
+from models.basemodel import BaseModel, db
+from models.krogerservice import Krogerservice
+from models.shopping_list import ShoppingList
+from models.at_risk_user import AtRiskUser
+from models.item import Item
+from datetime import datetime
+import requests
+
+class ShoppingLists(Resource):
+
+  def post(self,id):
+    request_data = request.json
+    at_risk_user = AtRiskUser.query.filter_by(at_risk_user_id=id).first()
+    if at_risk_user is None:
+      return {'data': { 'id': 'shoppinglist', 'attributes': {'error': "At risk user #{} not found".format(id)} } }, 400
+    else:
+      shopping_list = ShoppingList(at_risk_user_id=at_risk_user.id, status=request_data['status'], created_date=datetime.now())
+      db.session.add(shopping_list)
+      db.session.commit()
+      for item in request_data["items"]:
+        db.session.add(Item(shopping_list_id=shopping_list.id, **item)) 
+      db.session.commit()
+      return {'message': 'new shopping list created for user #{}'.format(id)}, 200
+
+  def get(self,id):
+    at_risk_user = AtRiskUser.query.filter_by(at_risk_user_id=id).first()
+    if at_risk_user is None:
+      return {'data': { 'id': 'shoppinglist', 'attributes': {'error': "At risk user not found"} } }, 400
+    else:
+      shopping_list = at_risk_user.shopping_lists[-1]
```

Comment:
need to add conditional here in case no `shopping_list` matches the `at_risk_user`